src/space.c

```diff
@@ -1475,6 +1475,15 @@ void space_update( const double dt )
 }
 
 
+/**
+ * @brief returns wether we're just simulating.
+ */
+unsigned int space_isSimulation( void )
```

Comment:
Don't think this is necessary if instead of using `ntime_t now;` you use the `dt` directly (which is only >0. when simulating).

Reply:
The problem is that in simulation, the tracks don't increment their age, so they don't vanish. When the player takes off, there are lines all over the space.
I think using `dt` won't change that.

Reply:
It should work, because that's how all the pilot timers and weapon timers work. Most of the timers are implemented by setting the time to live and decrementing. When the timer is `<0` it is over. Wouldn't something like that work?